css/index.css

```diff
@@ -1,3 +1,283 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+	
+
+
+.container{
+	max-width: 878px;
+	width: 100%;
+    margin: 0 auto;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 90%;
+        hight: auto;
+    }
+}
+
+@media (max-width: 500px) {
+    .containter {
+        width: 100%;
+        height: auto;
+    }
+}
+
+header {
+	display: inline-block;	
+}
+
+header nav{
+	padding: 20px 0;
+}
+
+@media (max-width: 800px) {
+    header nav{
+    display: flex;
+    flex-wrap: wrap-reverse;
+    justify-content: center;
+    }
+}
+
+@media (max-width: 500px) {
+    header nav {
+    flex-direction: column;
+    padding-top: 0;
+    }
+}
+
+
+header nav a{	
+	margin-right: 61px;
```

Comment:
I would take the `margin-right` off of these a tags and instead use flexbox to space them out. At certain widths, the A tags end up wrapping and looking a little weird (try looking at the page at 850px or 630px)